app/Http/Controllers/CompetitionsController.php

```diff
@@ -16,7 +25,8 @@ class CompetitionsController extends Controller
      */
     public function index()
     {
-        return view('competitions.index');
+        $competitions = Competition::all();
+        return view('competitions.index')->withCompetitions($competitions);
```

Comment:
It's a Taylor-esque "magic" alias for `view()->with(['competitions' => $competitions]`. Personally, I think I prefer `view()->with(compact('competitions'))` for consistency but it's all good.